src/main/java/com/inventorysystem/inventory/dto/Product.java

```diff
@@ -3,10 +3,10 @@
 
 public @Data
 class Product {
-    private int ProductID;
-    private String ProductName;
-    private String ProductDesc;
-    private int VendorID;
-    private double Price;
-    private int Quantity;
+    private int productID;
+    private String productName;
+    private String productDesc;
+    private int vendorID;
+    private double price;
+    private int quantity;
```

Comment:
Class variables/fields must be lowercase. [Reference](https://www.codejava.net/coding/10-java-core-best-practices-every-java-programmer-should-know#NamingConvention)

Reply:
yes, good.